src/style.css

```diff
@@ -0,0 +1,19 @@
+.block {
+  height: 100px;
+}
+
+.red {
+  background-color: red;
+}
+
+.green {
+  background-color: green;
+}
+
+.blue {
+  background-color: blue;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
move body to the top of the document, it's usually written there

Reply:
Done